app/src/main/res/values/strings.xml

```diff
@@ -117,10 +117,10 @@
     <string name="recent_tabs_show_all_content_description_2">Show all recent tabs button</string>
     <!-- Title for showing a group item in the 'Jump back in' section of the new tab
         The first parameter is the search term that the user used. (for example: your search for "cat")-->
-    <string name="recent_tabs_search_term">Your search for \"%1$s\"</string>
+    <string name="recent_tabs_search_term" moz:RemovedIn="105" tools:ignore="UnusedResources">Your search for \"%1$s\"</string>
     <!-- Text for the number of tabs in a group in the 'Jump back in' section of the new tab
         %d is a placeholder for the number of sites in the group. This number will always be more than one. -->
-    <string name="recent_tabs_search_term_count_2">%d sites</string>
```

Comment:
Just for my own education: what's the workflow like using this `moz:RemovedIn` flag? Will it get removed during translations?

Reply:
It's a handy little tag @Amejia481 made that makes it easier to track down the strings to be removed three releases later (I believe it's three). As far as I know, it's still a manual removal process, and I think @gabrielluong has done some of them.

Reply:
found a wiki article https://github.com/mozilla-mobile/fenix/wiki/Removing-strings

Reply:
We mark the version number that the string is removed in. We will remove then remove the string when the string is no shipped in any of the versions of Fenix (Nightly, Beta, Release) to remove any breakage where we might end up uplifting string changes (including string that were removed in Nightly) into Beta, but the string might still be used in Beta whereas it is no longer used in Nightly. It is fairly documented in https://github.com/mozilla-mobile/fenix/wiki/Removing-strings.